### ISSUE TITLE
[question] slice and dice

### ISSUE THREAD

**Issue by V-lpen** [Flomik10002/DelightfulCreatorsFabric#10]
You mention this mods methods are different than Slice and Dice, should Delightful Creators replace Slice n Dice,  or do they work together?

**Comment by Flomik10002** [comment]
> You mention this mods methods are different than Slice and Dice, should Delightful Creators replace Slice n Dice,  or do they work together?

My mod uses something other than Slice and Dice, but I don’t recommend installing it together. I advise you to install only my mod, since the crafts in my mod are complicated and will be interchangeable, which is not advisable

**Comment by V-lpen** [comment]
I went ahead and replaced it and i love the stuff this mod adds! Thank you! 

Though I do wish there was a way I could still craft AND use these recipes. 
Or at the very least manual sequenced assembly without a deployer. just in case a player doesnt wish to use create in the modpack. 
(Edit: I just moved all the recipes into a datapack not in the farmersdelight namespace and I got that result ignore me lol)
